haystack/components/generators/utils.py

```diff
@@ -3,9 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import json
-from typing import Any, Dict, List
-
-from openai.types.chat.chat_completion_chunk import ChoiceDeltaToolCall
+from typing import Dict, List
```

Comment:
We've updated both utility functions in this file to use the new StreamingChunk fields instead of relying on specific values in the metadata.